build.sbt

```diff
@@ -3,18 +3,16 @@ import com.gu.riffraff.artifact.RiffRaffArtifact.autoImport.{riffRaffArtifactRes
 val compilerOptions = Seq(
   "-deprecation",
   "-Xfatal-warnings",
-  "-encoding", "UTF-8",
-  "-target:jvm-1.8"
```

Comment:
As mentioned [on documentation](https://docs.scala-lang.org/overviews/compiler-options/index.html) `target` has been deprecated and replaced with `release`, however:
- `8` is the default for `target`
- We have moved to Java 11 as our default